doc/news/DM-38338.feature.rst

```diff
@@ -0,0 +1 @@
+Update maintel/track_target_and_take_image_gencam_.py to allow taking images with multiple cameras.
```

Comment:
I suggest wrapping the script name in "``", e.g.:

```
Update ``maintel/track_target_and_take_image_gencam.py`` to allow taking images with multiple cameras.
```

Also, there's and extra _ in the name